data/json/materials.json

```diff
@@ -1752,6 +1752,26 @@
     "cut_dmg_verb": "chipped",
     "burn_products": [ [ "pebble", 1 ] ]
   },
+  {
+    "type": "material",
+    "id": "monolith",
```

Comment:
Are you wanting all the artifacts to use this material or just random artifacts, or even only some random artifacts?

Reply:
I would like to convert the other artifacts to this material, and will if it's alright with the devs.